docs/remote/attach-container.md

```diff
@@ -0,0 +1,79 @@
+g---
```

Comment:
There's an extra "g" here that I assume will cause trouble once it lands in the docs site